lib/megaLogger.js

```diff
@@ -311,34 +284,34 @@
 
                 try {
                     text = JSON.stringify(noColorMsg);
-                } catch(e) {
+                }
+                catch (e) {
                     text = noColorMsg.join(' ');
                 }
 
                 var fn = "log";
-                if(level == MegaLogger.LEVELS.DEBUG) {
-                    fn = "debug"
-                } else if(level == MegaLogger.LEVELS.LOG) {
-                    fn = "log"
-                } else if(level == MegaLogger.LEVELS.INFO) {
-                    fn = "info"
-                } else if(level == MegaLogger.LEVELS.WARN) {
-                    fn = "warn"
-                } else if(level == MegaLogger.LEVELS.ERROR) {
-                    fn = "error"
+                if (level === MegaLogger.LEVELS.DEBUG) {
+                    fn = "debug";
+                } else if (level === MegaLogger.LEVELS.LOG) {
+                    fn = "log";
+                } else if (level === MegaLogger.LEVELS.INFO) {
+                    fn = "info";
+                } else if (level === MegaLogger.LEVELS.WARN) {
+                    fn = "warn";
+                } else if (level === MegaLogger.LEVELS.ERROR) {
+                    fn = "error";
                 }
```

Comment:
NB: all this block makes no sense, since this is only reached when `level === MegaLogger.LEVELS.ERROR`